source/PlayerInfo.cpp

```diff
@@ -808,19 +808,22 @@ const vector<shared_ptr<Ship>> &PlayerInfo::Ships() const
 // Returns a mapping of ships to the reason their flight check failed.
 map<const shared_ptr<Ship>, vector<string>> PlayerInfo::FlightCheck() const
 {
+	// Stores the relations between ships and docks.
+	// True: The ship is docked
+	// False: The ship requires a dock but is not docked
+	// Missing: The ship doesn't require a dock, and is therefore not docked
+	auto dockedShips = map<shared_ptr<Ship>,bool>{};
```

Comment:
Best this way, so we don't initialise one to transfer it later.

Also no need to initialize it as empty it does it by itself.
```suggestion
	map<shared_ptr<Ship>, bool> dockedShips;
```